lib/ThinEvent.php

```diff
@@ -3,12 +3,13 @@
 namespace Stripe;
 
 /**
- * @property string             $id Unique identifier for the event.
- * @property string             $type
- * @property string             $created
- * @property null|string        $context
- * @property null|RelatedObject $related_object
- * @property null|Reason        $reason
+ * @property string             $id       Unique identifier for the event.
+ * @property string             $type     The type of the event.
+ * @property string             $created  Time at which the object was created.
+ * @property null|string        $context  Authentication context needed to fetch the event or related object.
+ * @property null|RelatedObject $related_object // TODO Check if this can be removed and fix the tests
+ * @property null|Reason        $reason   Reason for the event.
```

Comment:
Are reason and livemode available in ThinEvent? I thought it was only the above five.

Reply:
I'll remove those.